src/storage/src/sst.rs

```diff
@@ -296,6 +301,8 @@ pub struct FileMeta {
     pub time_range: Option<(Timestamp, Timestamp)>,
     /// SST level of the file.
     pub level: Level,
+    /// Size of the file.
+    pub file_size: u64,
```

Comment:
[compatibility]: Better to use `pub file_size: Option<u64>` instead, otherwise when upgrade to this revision, users with existing SST files will face panics, since these SST files do not have this field in manifest.

Reply:
Could we support `Default` and use 0 as its default value?

Reply:
> @v0y4g3r Nice catch.
> 
> But I agree with @evenyag to use 0 as default value instead of `Option<u64>` for more convenient access with `file_size`. And We can add `#[serde(default)]` on `file_size` to use 0 as default value for compatibility when deserialize.
> 
> WDYT ?

I'm ok with it.

Reply:
Could we just derive default for the whole struct? Then we don't need to add `#[serde(default)]` for each field newly added.
```rust
#[derive(Clone, Debug, Default, PartialEq, Eq, Hash, Serialize, Deserialize)]
#[serde(default)]
pub struct FileMeta {
// ...
}
```

Reply:
Had a second thought and I think is ok. I was afraid there will be some special compatibility logic for newly added field, but it seems with or without `default` on struct both can use `Option` to handle.